### PR TITLE
Added english documentation for extended TTL syntax

### PR DESCRIPTION
**Pull request by excitoon**
I hereby agree to the terms of the CLA available at: https://yandex.ru/legal/cla/?lang=en

Changelog category (leave one):
- Documentation

### CODE REVIEW

File: docs/en/operations/table_engines/mergetree.md [hunk 1]
```diff
@@ -70,11 +70,13 @@ For a description of parameters, see the [CREATE query description](../../query_
 
     If a sampling expression is used, the primary key must contain it. Example: `SAMPLE BY intHash32(UserID) ORDER BY (CounterID, EventDate, intHash32(UserID))`.
 
-- `TTL` — An expression for setting storage time for rows.
+- `TTL` — A list of rules specifying durations of storage of rows and defining logic of automatic moving parts between disks and volumes.
```
Comment (blinkov) [inline docs/en/operations/table_engines/mergetree.md]:
```suggestion
- `TTL` — A list of rules specifying storage duration of rows and defining logic of automatic parts movement between disks and volumes.
```
also probably needs a link to docs on how to specify disks/volumes in configuration

File: docs/en/operations/table_engines/mergetree.md [hunk 2]
```diff
@@ -70,11 +70,13 @@ For a description of parameters, see the [CREATE query description](../../query_
...
 
-    It must have one `Date` or `DateTime` column as a result. Example:
+    Expression must have one `Date` or `DateTime` column as a result. Example:
     `TTL date + INTERVAL 1 DAY`
 
+    Type of the rule `DELETE|TO DISK 'xxx'|TO VOLUME 'xxx'` specified an action which is to be done with the part if expression is satisfied (reaches current time): deletion of expired rows, moving of part (if expression satisfies for all rows of a part) to specified disk (`TO DISK 'xxx'`) or volume (`TO VOLUME 'xxx'`). Default type of the rule is deletion (`DELETE`). Only one `DELETE` expression can be specified in the list of rules.
```
Comment (blinkov) [inline docs/en/operations/table_engines/mergetree.md]:
```suggestion
    Type of the rule `DELETE|TO DISK 'xxx'|TO VOLUME 'xxx'` specifies an action to be done with the part if the expression is satisfied (reaches current time): removal of expired rows, moving a part (if expression is satisfied for all rows in a part) to specified disk (`TO DISK 'xxx'`) or to volume (`TO VOLUME 'xxx'`). Default type of the rule is deletion (`DELETE`). List of multiple rules can specified, but there should be no more than one `DELETE` rule.
```
probably needs a mention of what happens if all rows in a part satisfy multiple rules

File: docs/en/operations/table_engines/mergetree.md [hunk 3]
```diff
@@ -371,9 +373,11 @@ Reading from a table is automatically parallelized.
 
 Determines the lifetime of values.
 
-The `TTL` clause can be set for the whole table and for each individual column. If both `TTL` are set, ClickHouse uses that `TTL` which expires earlier.
+The `TTL` clause can be set for the whole table and for each individual column. Table-level TTL can also specify logic of automatic move of data between disks and volumes.
+
+Expressions must evaluate to [Date](../../data_types/date.md) or [DateTime](../../data_types/datetime.md) data type.
 
-To define the lifetime of data, use expression evaluating to [Date](../../data_types/date.md) or [DateTime](../../data_types/datetime.md) data type, for example:
+To define the lifetime of data in a column, for example:
```
Comment (blinkov) [inline docs/en/operations/table_engines/mergetree.md]:
```suggestion
Example:
```

File: docs/en/operations/table_engines/mergetree.md [hunk 4]
```diff
@@ -430,6 +434,18 @@ ALTER TABLE example_table
 
 When data in a table expires, ClickHouse deletes all corresponding rows.
```
Comment (blinkov) [inline docs/en/operations/table_engines/mergetree.md]: this sentence is kind of incorrect now and probably could be just removed

File: docs/en/operations/table_engines/mergetree.md [hunk 5]
```diff
@@ -430,6 +434,18 @@ ALTER TABLE example_table
...
 
+For a table one can set an expression for deletion of expired rows, and expressions which if satisfied will trigger automatic move of part to [another disk or volume](#table_engine-mergetree-multiple-volumes). When rows in the table expire, ClickHouse deletes all corresponding rows.
```
Comment (blinkov) [inline docs/en/operations/table_engines/mergetree.md]:
```suggestion
Table can have an expression for deletion of expired rows, and multiple expressions for automatic move of parts between [disks or volumes](#table_engine-mergetree-multiple-volumes). When rows in the table expire, ClickHouse deletes all corresponding rows. For parts moving feature, all rows of a part must satisfy the movement expression criteria.
```

**Review by blinkov — CHANGES_REQUESTED** [review]

File: docs/en/operations/table_engines/mergetree.md (hunk 4 above)
Comment (excitoon) [reply docs/en/operations/table_engines/mergetree.md]: That's right.

**Pull request merged**